e2e-tests/cucumber-messages.ndjson

```diff
@@ -0,0 +1,64 @@
+{"testRunStarted":{"timestamp":{"seconds":1676286317,"nanos":602000000}}}
```

Comment:
Ignore this file?

Reply:
Remove

Reply:
Yes, it should not have been added to the repo.
It is automatically created by the cucumber framework and will be ignored by git.